src/libraries/Microsoft.XmlSerializer.Generator/src/GenerateThisAssemblyCs.targets

```diff
@@ -4,6 +4,8 @@
   </ItemGroup>
 
   <Target Name="GenerateThisAssemblyCs"
+          Inputs="$(MSBuildThisFileFullPath)"
+          Outputs="$(IntermediateOutputPath)ThisAssembly.cs"
```

Comment:
Nit: Can we put this in a property so that if we ever update the name we don't miss the Outputs and break incremental?

Reply:
This can't be extracted into a property unless it would we inside a target as the IntermediateOutputPath isn't available at this point as a property. Introducing a separate target just for defining this property seems overkill so it's probably better to leave as is.

Reply:
Makes sense.